projects/addon-editor/tokens/files-loader.ts

```diff
@@ -0,0 +1,23 @@
+import {InjectionToken} from '@angular/core';
+import {
+    TuiEditorAttachedFile,
+    TuiEditorAttachOptions,
+} from '@taiga-ui/addon-editor/interfaces';
+import {TuiHandler} from '@taiga-ui/cdk';
+import {Observable, of} from 'rxjs';
+
+export const TUI_ATTACH_FILES_OPTIONS: InjectionToken<TuiEditorAttachOptions> =
+    new InjectionToken<TuiEditorAttachOptions>(
+        `[TUI_ATTACH_FILES_OPTIONS]: files loader handler`,
+        {factory: () => ({accept: `*/*`, multiple: true})},
+    );
+
+export const TUI_ATTACH_FILES_LOADER: InjectionToken<
+    TuiHandler<File[], Observable<TuiEditorAttachedFile[]>>
+> = new InjectionToken<TuiHandler<File[], Observable<TuiEditorAttachedFile[]>>>(
+    `[TUI_ATTACH_FILES_LOADER]: files loader handler`,
+    {
+        factory: () => files =>
+            of(files.map(file => ({name: file.name, link: file.name}))),
```

Comment:
This doesn't do anything, just the names. I don't think it makes sense for default value. How about we instead enable attach file tool if the loader was provided?

Reply:
fixed